src/index.html

```diff
@@ -4,14 +4,166 @@
     <meta charset="UTF-8">
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+      content="width=device-width, user-scalabel=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="POST">
+      <fieldset class="form-group user-info_group"> 
+        <legend>Personal information</legend>
+        <label class="form-field">
+          Surname:
+          <input
+            type="text"
+            name="surname"
+            minlength="2"
+            autocomplete="off"
+            required
+          >
+        </label>
+        <label class="form-field">
+          Name:
+          <input 
+            type="text"
+            name="name"
+            minlength="2"
+            autocomplete="off"
+            required
+          >
+        </label>
```

Comment:
```suggestion
          <input
            type="text"
            name="surname"
            minlength="2"
            autocomplete="off"
            required
          >
        </label>
        
        <label class="form-field">
          Name:
          <input 
            type="text"
            name="name"
            minlength="2"
            autocomplete="off"
            required
          >
        </label>
```
Add empty lines between siblings here as well

Reply:
done